securedrop/crypto_util.py

```diff
@@ -50,7 +50,7 @@ def do_runtime_tests():
 gpg = gnupg.GPG(binary='gpg2', homedir=config.GPG_KEY_DIR)
 
 # map code for a given language to a localized wordlist
-language2words = {}  # type: Dict[Text,List[Text]]
```

Comment:
this fixes the previous commit. If you would like to keep it as two commits, could you please move this to the previous commit ? Otherwise squashing into one is good because the two commits are closely related.